R/Module-PatientLevelPrediction.R

```diff
@@ -130,7 +130,7 @@ PatientLevelPredictionModule <- R6::R6Class(
         csvFolder = resultsFolder,
         connectionDetails = resultsConnectionDetails,
         databaseSchemaSettings = databaseSchemaSettings,
-        modelSaveLocation = file.path(resultsFolder, "dbmodels"),
+        modelSaveLocation = file.path(resultsFolder, "models"),
```

Comment:
This code was taken from https://github.com/OHDSI/PatientLevelPredictionModule/blob/main/Main.R#L197-L204. Tagging @jreps to confirm she has no concerns otherwise this looks fine to me.